DotNetRu.Clients.Portable/ViewModel/SpeakersViewModel.cs

```diff
@@ -22,16 +23,19 @@ public SpeakersViewModel(INavigation navigation)
         {
         }
 
-        public ObservableRangeCollection<SpeakerModel> Speakers { get; } =
-            new ObservableRangeCollection<SpeakerModel>();
+        public ObservableRangeCollection<Grouping<string, SpeakerModel>> _speakers = new ObservableRangeCollection<Grouping<string, SpeakerModel>>();
```

Comment:
_speakers is against code style, should be just speakers
Please use Resharper + StyleCop for Resharper